examples/cosign/README.md

```diff
@@ -0,0 +1,161 @@
+# Cosign Pipeline
+
+This pipeline depicts how to build a Golang project from source, package it in a
+container using `ko` and verify that the processed was not tampered with.
+
+## How to run this demo
+
+Execute the following commands from the root of this repository:
+
+```bash
+# Only if a cluster is needed.
+make setup-minikube
+
+# Setup tekton w/ chains
+make setup-tekton-chains tekton-generate-keys
+
+# Run a new pipeline.
+make example-cosign
+# Or re-run the last one.
+# tkn pipeline start ko -L
+
+# Wait until it completes.
+tkn pr logs --last -f
+
+# Ensure it has been signed.
+tkn tr describe --last -o jsonpath='{.metadata.annotations.chains\.tekton\.dev/signed}'
```

Comment:
I don't think this currently works, since the `build-with-ko` task is not the last task run.
We may need to use something like the following to get the task run that produced the image that was signed.
```
export TASK_RUN=$(tkn pr describe --last -o json | jq -r '.status.taskRuns | keys[] as $k | {"k": $k, "v": .[$k]} | select(.v.status.taskResults[]?.name | match("IMAGE_URL$")) | .k')
tkn tr describe "${TASK_RUN}" -o jsonpath='{.metadata.annotations.chains\.tekton\.dev/signed}'
```

Reply:
Good catch! implementing that right now.